networking/manta-net.sh

```diff
@@ -817,6 +817,24 @@ function handle_manta_nics
 	done
 }
 
+#
+# If this Triton is setup to use net-boot config (i.e. networking.json, fabics)
+# then we must update /mnt/usbkey/boot/networking.json with the manta
+# nic tag info.
+#
+function update_headnode_netboot_config
+{
+	if /usr/lib/sdc/net-boot-config --enabled; then
+		sdc-usbkey mount
+		sdc-login -l dhcpd /opt/smartdc/booter/bin/hn-netfile \
+		    > /mnt/usbkey/boot/networking.json
+		sdc-usbkey unmount
```

Comment:
Should this not unmount if it didn't mount? e.g. if it were already mounted when `sdc-usbkey mount` ran?

Reply:
Done.  I re-ran on nightly-1:

```
[root@headnode (nightly-1) ~]# sdc-usbkey mount
/mnt/usbkey
[root@headnode (nightly-1) ~]# sdc-usbkey status
mounted

[root@headnode (nightly-1) /var/tmp/networking]#     ./manta-net.sh /var/tmp/manta-setup-netconfig.json
...
+ update_headnode_netboot_config
+ local usbkey_status
+ /usr/lib/sdc/net-boot-config --enabled
++ sdc-usbkey status
+ usbkey_status=mounted
+ [[ mounted == \m\o\u\n\t\e\d ]]
+ sdc-login -l dhcpd /opt/smartdc/booter/bin/hn-netfile
+ [[ mounted == \m\o\u\n\t\e\d ]]
+ warn 'Successfully updated /mnt/usbkey/boot/networking.json'
+ local 'msg=Successfully updated /mnt/usbkey/boot/networking.json'
+ [[ -z Successfully updated /mnt/usbkey/boot/networking.json ]]
+ echo 'manta-net.sh: Successfully updated /mnt/usbkey/boot/networking.json'
manta-net.sh: Successfully updated /mnt/usbkey/boot/networking.json
...
+ exit 0

[root@headnode (nightly-1) /var/tmp/networking]# sdc-usbkey status
mounted
```